### ISSUE TITLE
Remove "fight like a girl" dialogue

### ISSUE THREAD

**Issue by alexrovw** [endless-sky/endless-sky#9662]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I think we should remove the somewhat misogynist bits of dialogue from this conversation with Katya.

<img width="565" alt="Screenshot 2024-01-12 at 10 37 07 PM" src="https://github.com/endless-sky/endless-sky/assets/98854112/42754ba2-56f9-4fb9-9993-18b2e8aac6a9">

Reasoning:
1. It's not appropriate for Katya to accuse the player of fighting like a girl, when that phrase has a connotation that belittles women. [Certain groups](https://www.fightlikeagirlclub.com) have been reclaiming this phrase, but Katya is obviously not using it in a positive way.
2. This phrase is not merely an insult but a sexist comment. We don't include any other such discriminatory language in the game, and this should be no exception.
3. First Last is gender neutral/ambiguous and should support all genders of player. It doesn't make logical sense for the retort, "well you ARE a girl!" to exist.

### Steps to Reproduce

1. Begin new pilot
2. Fast forward game date to around August 1st 3014, after the Geminus bombing.
3. Visit a Free Worlds spaceport to initiate dialogue with Katya
4. See choices in the screenshot above

### Expected Behavior

Dialogue should be gender neutral and respectful of the actual human being playing the game

### Screenshots

Screenshot visible in above description

### Operating System

All

### Game Source

Built from source

### Game Version

0.10.5

**Comment by Amazinite** [comment]
> First Last is gender neutral/ambiguous and should support all genders of player. It doesn't make logical sense for the retort, "well you ARE a girl!" to exist.

There's a difference between narrative and player choice. This response is a player choice, therefore it's up to the player to characterize their own character. The narrative of the game and the dialog of other characters should always remain neutral, though.

**Comment by alexrovw** [comment]
I'd be interested to hear the thoughts of a woman contributor. This line was written by MZ 10 years ago, and I don't think it aged well.

**Comment by Amazinite** [comment]
Quoting a comment of yours from the PR.
> This line was written by MZ 10 years ago, and I think it was probably more socially acceptable then than it is now.

If we can grant that there was no issue with this line 10 years ago but claim that there's an issue now, then we're effectively just dealing with the [euphemism treadmill](https://en.wikipedia.org/wiki/Euphemism#Lifespan). I'd rather not police the game for lines that were fine at the time but for one reason or another have become an issue for certain parts of certain cultures. What's to stop the alternative we change to from becoming an issue another 10 years down the line? The only way to win is to not play the game.

**Comment by diodelass** [comment]
I do think that this dialog reads as a bit sexist, but I think cleaning it up could be very easy.

To be clear, I _don't_ think that giving the player a sexist dialog option, nor even making Katya's reply somewhat sexist too, is necessarily problematic. A story can have prejudiced characters without itself being prejudiced, etc etc. But the available dialog options for the player here do feel like they're assuming too much about the player's gender (namely that they are male), and I also feel that it would suit Katya's character better to put just a slightly finer point on her rebuttal.

Some small changes that I think would improve this dramatically:
- Slightly changing Katya's reply to "You ARE a girl!" to have her open with "No, I'm a woman" - deflecting the sexism and contextualizing her initial "fight like a girl" accusation as more of a maturity thing, and less of a gender thing (and retaining the good bit about fighting like a she-badger).
- Giving the player an option to say "I AM a girl!", with Katya responding something like "Well, I'd taken you for a woman, but clearly I was wrong!".
- Giving the player an option to say "Like a girl? Why, thank you!" - backhandedly accepting Katya's accusation as if it were a compliment, with her responding something similar to the above.

In particular, I really like the idea of the game allowing the player to self-determine their gender in this interaction and have Katya just go along with whatever they say they are. There's no need to track what gender the player identifies themselves with in their save file for the future either, of course; if for some reason it needs to come up again, they can just pick whatever gendered dialog options may suit them at that time, too. Doing things this way would make it clear that the game's actual stance on gender issues is very agnostic and hands-off, allowing many different possible interactions - including the one in which the player is misogynistic toward Katya - without implicitly endorsing any of them.

Finally, I just want to say that I wholeheartedly agree with @MCOfficer on the subject of whether this needs to change in the first place - the game is in development now, and that means players will be judging it with current standards. If I recommend this game to a family member and they come back to me complaining about a scene being sexist, I can't brush that off with "well, it was a product of its time" nonsense, because as far as anyone is concerned, "its time" is now. I mean, if you read a novel published this year and it had something gross about women repeated uncritically in it, nobody would be impressed by the author saying something like "oh don't worry about that, this book took 40 years to write and I did that bit back when it was okay to say that". Isn't that what editors are for?

I know it can feel frustrating to have to keep updating long-running projects like this to fix language that seemingly "becomes offensive" over time, but remember that these are not cases of previously-benign language "becoming" offensive, but ones in which more people are becoming aware of how much damage the language in question has _always_ caused. And I really feel like "the winning move is not to play" couldn't be more wrong in this case. It is simply not possible to "not play"; we have been "playing" ever since this dialog was published in the game. To continue forward without doing anything about this sort of thing is to play and to lose.

For what it's worth, I think Endless Sky has done an admirable job of exploring all kinds of interesting social issues without embarrassing itself with outdated language or ideas, so I don't really see any risk of a "slippery slope" in the future where a constantly increasing number of things need to be changed to suit evolving sensibilities. And as things go, this particular dialog isn't especially egregious either, and I feel a bit silly for the wall of text I've just written about it. But it also seems silly that there's opposition to changing something so small.

**Comment by Hecter94** [comment]
I believe there has been enough discussion on this topic to warrant reopening it. Diodelass has offered some valuable suggestions for potential changes to the dialogue that appear to be objective improvements. 
I'd like to keep these suggestions in an open issue so we don’t lose them among the other closed issues.

**Comment by tibetiroka** [comment]
Yeah, I think we should change the responses in that section.

**Comment by Saugia** [comment]
I think what we could do is add another character choice like, "I take that as a compliment".

**Comment by xobes** [comment]
How about this?

~~First of all, if~~ **If** this gun were loaded, your brains would be all over the sidewalk ~~And second you fight like a girl.~~ I have no use for hotheads like you." She stalks off, leaving you very confused.  It would appear you've just failed some sort of test.

  "Oh yeah? Well, ~~you ARE a girl!~~ you snuck up on me! You think that makes you tough?"

She turns back around, with a fearsome gleam in her eyes, and walks up to you until your faces are only inches apart.  "Yeah, ~~but~~ well, I don't fight like ~~one~~ **a clown**," she says. "I fight like a ~~she-~~ badger."

"A ~~she-~~ badger?"

"Fearsome creatures." ...

**Comment by xobes** [comment]
Thoughts?  Hate it? Like it? Should I PR it?

**Comment by Amazinite** [comment]
I prefer Saugia's approach. Also the she-badger line is really funny, I'd rather not toss it.

**Comment by Crustaceanus** [comment]
Yeah the dialogue is a bit sexist, but I think that going further into it and adding more dialog options or retorts is a bit silly and definitely a lot more work.. Why not just change it to a simple line similar to, "You fight like a rookie." or "You fight like someone who learned from a manual."

Its a simple fix really, just rephrase a bit and nobody will have issues.. I'm sure that someone else can come up with a line that makes more sense, but I segest that we simply change the allegory, rather then add a bunch of responses..
